src/analysis/retail/monk/mistweaver/modules/spells/Unison.tsx

```diff
@@ -0,0 +1,74 @@
+import { TALENTS_MONK } from 'common/TALENTS';
+import SpellLink from 'interface/SpellLink';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events, { HealEvent} from 'parser/core/Events';
+import { formatNumber } from 'common/format';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+
+class Unison extends Analyzer {
+healing: number = 0;
+overhealing: number = 0;
+healingFromJss: number = 0;
+
+constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(
+        TALENTS_MONK.UNISON_TALENT.id,
+    );
+    if(!this.active) {
+        return;
+    }
+    this.addEventListener(
+        Events.heal.by(SELECTED_PLAYER).spell(SPELLS.UNISON_HEAL),
+        this.unisonHeal,
+    );
+    if(this.selectedCombatant.hasTalent(TALENTS_MONK.SUMMON_JADE_SERPENT_STATUE_TALENT.id)) {
+        this.addEventListener(
+        Events.heal.by(SELECTED_PLAYER_PET).spell(SPELLS.UNISON_HEAL),
+        this.jssUnisonHeal,
+        );
+    }
+}
+
+unisonHeal(event: HealEvent) {
+    this.healing += (event.amount || 0 ) + (event.absorbed|| 0);
+    this.overhealing += event.overheal || 0;
+}
+
+jssUnisonHeal(event: HealEvent) {
+    this.healing += (event.amount || 0 ) + (event.absorbed || 0);
+    this.overhealing += event.overheal || 0;
+    this.healingFromJss += (event.amount || 0) + (event.absorbed || 0);
+}
+
+statistic() {
+    return (
+        <Statistic
+            position={STATISTIC_ORDER.OPTIONAL(14)}
+            size="flexible"
+            category={STATISTIC_CATEGORY.TALENTS}
+            tooltip={
+                <ul>
+                    <li>
+                        Healing from <SpellLink id={TALENTS_MONK.SUMMON_JADE_SERPENT_STATUE_TALENT.id} />: {formatNumber(this.healingFromJss)} 
+                    </li>
+                    <li>
+                        Healing from <SpellLink id={TALENTS_MONK.SOOTHING_MIST_TALENT.id} />: {formatNumber(this.healing - this.healingFromJss)}
+                    </li>
+                </ul>
+            }
+        >
+            <BoringSpellValueText spellId={SPELLS.UNISON_HEAL.id}>
```

Comment:
The spell value here also doesn't have to be the exact ID of the heal; you could use `TalentSpellText` with the talent value and it should be good enough. This is also fine though, just wanted to point it out.

Reply:
I did try that at first, but the talent and heal id values are different and `TalentSpellText` only takes the `Talent` object (I'm pretty sure) as a parameter. So unfortunately it would return the proper icon and Wowhead link but the text returned 'Unknown'.

Reply:
That might need to be something we look at in `TalentSpellText`, then. We can ignore that for this PR, I'll take a peek

Reply:
Can confirm, rewrote it using TalentSpellText and ran into the same issue